src/components/Cheatsheet.module.css

```diff
@@ -1,12 +1,12 @@
 .cheatsheet {
   height: auto !important;
   /* page height - navbar height - some spacing*/
-  max-height: calc(100vh - 76px - 1rem) !important;
+  max-height: calc(100% - 4.75rem - 1rem) !important;
```

Comment:
`100vh` causes issues on iOS since browser UI elements are part of the view port. Before, the cheatsheet would grow underneath the navbar. Now it doesn't anymore.